spec/support/dairy_data.rb

```diff
@@ -1,13 +1,13 @@
-Cheese = Struct.new(:id, :flavor, :fat_content, :source)
+Cheese = Struct.new(:id, :flavor, :origin, :fat_content, :source)
 CHEESES = {
-  1 => Cheese.new(1, "Brie", 0.19, 1),
-  2 => Cheese.new(2, "Gouda", 0.3, 1),
-  3 => Cheese.new(3, "Manchego", 0.065, "SHEEP")
+  1 => Cheese.new(1, "Brie", "France", 0.19, 1),
+  2 => Cheese.new(2, "Gouda", "Switzerland", 0.3, 1),
+  3 => Cheese.new(3, "Manchego", "Italy", 0.065, "SHEEP")
```

Comment:
How could I accept this!?

![image](https://cloud.githubusercontent.com/assets/2231765/12338241/bd091348-badc-11e5-9e35-9f66f1c67d6e.png)

Ok, just kidding :laughing: